### ISSUE TITLE
Infinite print

### ISSUE THREAD

**Issue by zhourui123** [SpongePowered/SpongeForge#3214]
**I am currently running**

- SpongeForge version: 1.12.2-2838-7.1.10     
- Forge version: 14.23.5.2838  
- Java version: 1.8.0_144, Oracle Corporation
- Operating System: debian   Linux (amd64) version 3.16.0-6-amd64

- Plugins/Mods:

armourers_workshop
baubles
thaumcraft
botania
extrabotany
slashblade
slashblade_addon
twilightforest
luckperms
spongycord
nucleus

**Issue Description**
Infinite print, you must kill the server thread.  There is no redstone about code.

https://gist.github.com/zhourui123/81da03f72b85d1dd00938e6a1a910cae

**Comment by dualspiral** [comment]
> SpongeForge version: 1.12.2-2838-7.1.10 

Yeah, no. That's old. Try again with 7.3.0, come back when you do so and tell us whether it's a problem or not then.
